CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## [Unreleased]
+
+### Added
+
+- MlEnabledCgeSlc, CoreCGEML, and CGEMLFileUtil Documentation for sphinx [#542](https://github.com/IN-CORE/pyincore/issues/542)
```

Comment:
Is this in the current PR?

Reply:
no, I think it came from another PR

Reply:
I have updated the PR. Thanks for this catch :)